pkg/datastore/datastore.go

```diff
@@ -308,3 +321,38 @@ func (ds *memoryDatastore) OperatorSourceHasUpdate(opsrcUID types.UID, metadata
 func (ds *memoryDatastore) GetAllOperatorSources() []*OperatorSourceKey {
 	return ds.rows.GetAllRows()
 }
+
+func (ds *memoryDatastore) DoesSourceExist(source string) bool {
+	for _, opSrc := range ds.GetAllOperatorSources() {
+		if source == opSrc.Name.Name {
```

Comment:
Do we want to potentially be case insensitive here? And probably not just here, but in general for the Source field?

Reply:
I don't see any negatives with the suggested chanage - worst case scenario, a user is confused why `commUnity-operators` worked.

Reply:
We should stick with the kube cadence here.

Reply:
This will be resolved as a part of https://github.com/operator-framework/operator-marketplace/pull/197#discussion_r295353705